core/plugins/org.polarsys.capella.core.ui.search/src/org/polarsys/capella/core/ui/search/CapellaSearchSettings.java

```diff
@@ -157,7 +163,7 @@ public IStatus validate() {
 
   public static Pattern createPattern(String textPattern, boolean isCaseSensitive, boolean isRegExSearch,
       boolean isWholeWord) {
-    return PatternConstructor.createPattern(textPattern, isRegExSearch, false, isCaseSensitive, isWholeWord);
+    return PatternConstructor.createPattern(textPattern, isRegExSearch, true, isCaseSensitive, isWholeWord);

```

Comment:
This fixes the fact that the String Matcher symbols were not used as intended during the search (* for any string, ? for a character etc).